components/brave_wallet_ui/common/hooks/nft-pin.ts

```diff
@@ -44,15 +47,17 @@ export function useNftPin () {
 
   // redux
   const isNftPinningFeatureEnabled = useSafeWalletSelector(WalletSelectors.isNftPinningFeatureEnabled)
-  const nftsPinningStatus = useUnsafePageSelector(PageSelectors.nftsPinningStatus)
-  const isAutoPinEnabled = useSafePageSelector(PageSelectors.isAutoPinEnabled)
-
+  
+ // queries
+ const { data: isAutoPinEnabled } = useGetAutopinEnabledQuery()
+ const { data: nftsPinningStatus } = useGetNftsPinningStatusQuery(undefined, { pollingInterval: POLLING_INTERVAL_IN_MS })
```

Comment:
Let's define the config object outside of the component.

something like (feel free to name this something else):
```suggestion
 const { data: nftsPinningStatus } = useGetNftsPinningStatusQuery(undefined, POLLING_CONFIG)
```